### ISSUE TITLE
Cannot start program in typescript-migration branch

### ISSUE THREAD

**Issue by ptantanis** [jwu910/check-it-out#509]
Hi,

I'm trying to help with typescript-migration but have a problem with set up local development. 

Problem:
npm run tsc command not generate .js file but print help message instead.
<img width="891" alt="image" src="https://user-images.githubusercontent.com/36023177/197374167-ec97fd87-61a3-4f03-b197-ed5d8c81c954.png">

step to reproduce:
1. clone project and check out `typescript-migration` branch
2. run `npm install`
3. run `npm run build`

expected behavior: get build js file of ts file in dist folder

I think tsconfig file is missing here. I tried add tsconfig file in root folder and tsc is working fine. 
result after add `tsconfig`
<img width="527" alt="image" src="https://user-images.githubusercontent.com/36023177/197374378-ca824401-4710-4ffa-9bb3-f50828326e17.png">

**Comment by jwu910** [comment]
Hi @ptantanis thanks for pointing this out and apologies for responding so late. The ts migration work has been on hold for quite a while and I figured I probabaly missed something. Not sure when I'll have time to pick it back up, but will double check to see if i can get the config committed at the very least. From my memory i recall getting stuck on a fairly large portion of the work that was taking a little longer than expected, which kinda blocked getting the community involved.

I'll keep this issue open until I can get that updated.

**Comment by jwu910** [comment]
@ptantanis are you still seeing an issue here? Made some updates to the typescript-migration branch since this issue was opened. We will close this soon if we don't hear back. Thanks!

**Comment by jwu910** [comment]
i'm going to leave this open, after pulling everything into the `typescript-migration` branch, i'm seeing issues running the app locally

The error above isn't 100% accurate anymore, but seeing the following mention in the logs

```
Object.defineProperty(exports, "__esModule", { value: true });
                      ^

ReferenceError: exports is not defined in ES module scope
This file is being treated as an ES module because it has a '.js' file extension```

Haven't had time to debug and figure out what configs need to change yet. Open to PRs to get this fixed.

**Comment by jwu910** [comment]
This is fixed by #535
